src/Tooltip/index.js

```diff
@@ -2,7 +2,7 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import includes from 'lodash.includes'
 
-import glossary from '../Documentation/glossary'
+import glossary from '../../static/docs/glossary'
```

Comment:
@shcheklein @iAdramelk notice that similarly to sidebar.js I moved glossary.js into static/docs/

Should we also move pages/ into static/ BTW?

Reply:
I would say `pages` looks like `src` to me. Probably we keep them in `pages` only because it's a Next convention or something? @iAdramelk should have a better idea.

`glossary` - makes sense to me.

Reply:
I do also think that correct place the for pages are inside the `src` folder and `gatsby` place theme there, but default place for `next` pages is root, and I can't find a way to change it in the `next`'s docs. But there is a lot of things that I can't find there, so it is possible that it is configurable is some way.

P. S. Maybe we should convert glossary for `.js` to `.json` if it is only data?

Reply:
I can't remember why but there was a reason we had to use a JS file for the glossary. I think having to do with formatting the md to 80 cols. Anyway, leaving pages/ in root since it's a Next thing and outside the docs engine.